src/promptflow/promptflow/executor/_errors.py

```diff
@@ -175,6 +175,15 @@ def __init__(self, line_number, timeout):
         )
 
 
+class ProcessCrashError(UserErrorException):
+    """Exception raised when single line execution timeout"""
```

Comment:
The name is crash error but the description is execution timeout?

Reply:
Oh, sorry, copied from others and fixed